src/cata_tiles.cpp

```diff
@@ -477,26 +485,44 @@ void cata_tiles::load_tilejson_from_file(JsonObject &config, int offset, int siz
 
 tile_type *cata_tiles::load_tile(JsonObject &entry, const std::string &id, int offset, int size)
 {
-    int fg = entry.get_int("fg", -1);
-    int bg = entry.get_int("bg", -1);
-    if (fg == -1) {
-        // OK, keep this value, indicates "doesn't have a foreground"
-    } else if (fg < 0 || fg >= size) {
-        entry.throw_error("invalid value for fg (out of range)", "fg");
-    } else {
-        fg += offset;
+    tile_type *curr_subtile = new tile_type();
+
+    if ( entry.has_array("fg") ) {
+        JsonArray fg_array = entry.get_array("fg");
+        while (fg_array.has_more()) {
+            curr_subtile->fg.push_back(fg_array.next_int());
+        }
+    } else if (entry.has_int("fg") ) {
+        curr_subtile->fg.push_back(entry.get_int("fg"));
     }
-    if (bg == -1) {
-        // OK, keep this value, indicates "doesn't have a background"
-    } else if (bg < 0 || bg >= size) {
-        entry.throw_error("invalid value for bg (out of range)", "bg");
-    } else {
-        bg += offset;
+
+    if ( entry.has_array("bg") ) {
+        JsonArray bg_array = entry.get_array("bg");
+        while (bg_array.has_more()) {
+            curr_subtile->bg.push_back(bg_array.next_int());
+        }
+    } else if (entry.has_int("bg") ) {
+        curr_subtile->bg.push_back(entry.get_int("bg"));
     }
-    tile_type *curr_subtile = new tile_type();
-    curr_subtile->fg = fg;
-    curr_subtile->bg = bg;
+
+    for (auto& fg : curr_subtile->fg) {
+        if (fg < 0 || fg >= size) {
+            entry.throw_error("invalid value for fg (out of range)", "fg");
```

Comment:
Need to either tolerate a value of -1 to indicate no background or foreground, or update all the tilesets to remove their negative values, a lot of them fail to load now.